vcd/resource_vcd_org_vdc.go

```diff
@@ -137,33 +144,33 @@ func resourceVcdOrgVdc() *schema.Resource {
 				Description: "Storage profiles supported by this VDC.",
 			},
 			"memory_guaranteed": &schema.Schema{
-				Type:        schema.TypeFloat,
-				Optional:    true,
-				ForceNew:    true,
-				Description: "Percentage of allocated memory resources guaranteed to vApps deployed in this VDC. For example, if this value is 0.75, then 75% of allocated resources are guaranteed. Required when AllocationModel is AllocationVApp or AllocationPool. Value defaults to 1.0 if the element is empty.",
+				Type:             schema.TypeString,
```

Comment:
Why are we changing the type from Float to String?

This change breaks compatibility.

Reply:
Please read explanation I did for Dainius above.

Reply:
The explanation above is about the .ID field. 
I am talking about changing the type of a field that will break existing Terraform scripts.

Reply:
Copying here:

Reply:
This one changed